components/docker-up/docker-up/main.go

```diff
@@ -133,7 +133,11 @@ func runWithinNetns() (err error) {
 	// configure docker0 MTU (used as control plane, not related to containers)
 	args = append(args, fmt.Sprintf("--network-control-plane-mtu=%v", netIface.Attrs().MTU))
 
+	// cmp. ENT-77: This allows us to run dockerd >= 26.1
+	os.Setenv("DOCKER_TEST_RO_DISABLE_IPV6", "true")
```

Comment:
Before updating to 26.1, we should check if a fix for handling extended attributes has landed, too: https://linear.app/gitpod/issue/ENT-77/use-docker-ce-v26-again-we-downgraded-to-2409#comment-8b074761

Reply:
Ah, I was not clear on how and if those are related, so wanted to try it out :eyes:

Reply:
Makes total sense.

The TL;DR is, we downgraded for two reasons:
1. the ipv6 issue
2. extended attributes causing some images to fail running

Reply:
@kylos101 Could you review this? I added detailed steps under "how to test" - or you trust me it works :wink: 

I think this is a good step forward to get rid of this blocker. :+1:

Reply:
Testing. 

(this seems good, it would at least prepare workspaces, so that customers who wish to upgrade to v26 are not blocked)

Reply:
Works as advertised:
```
gitpod /workspace/gitpod (main) $ docker run curlimages/curl:8.1.2 -v -k https://example.com
Unable to find image 'curlimages/curl:8.1.2' locally
8.1.2: Pulling from curlimages/curl
8a49fdb3b6a5: Pull complete 
c452a6fc1daa: Extracting [==================================================>]  6.793MB/6.793MB
4ca545ee6d5d: Download complete 
docker: failed to register layer: lsetxattr user.overlay.origin /bin: operation not supported.
See 'docker run --help'.
gitpod /workspace/gitpod (main) $ docker version
Client: Docker Engine - Community
 Version:           26.1.4
 API version:       1.45
 Go version:        go1.21.11
 Git commit:        5650f9b
 Built:             Wed Jun  5 11:28:57 2024
 OS/Arch:           linux/amd64
 Context:           default

Server: Docker Engine - Community
 Engine:
  Version:          26.1.4
  API version:      1.45 (minimum version 1.24)
  Go version:       go1.21.11
  Git commit:       de5c9cf
  Built:            Wed Jun  5 11:28:57 2024
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.6.33
  GitCommit:        d2d58213f83a351ca8f528a95fbd145f5654e957
 gitpod:
  Version:          1.1.9
  GitCommit:        v1.1.9-0-gccaecfcb
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
gitpod /workspace/gitpod (main) $ docker run hello-world
Unable to find image 'hello-world:latest' locally
latest: Pulling from library/hello-world
c1ec31eb5944: Pull complete 
Digest: sha256:d1b0b5888fbb59111dbf2b3ed698489c41046cb9d6d61743e37ef8d9f3dda06f
Status: Downloaded newer image for hello-world:latest

Hello from Docker!
This message shows that your installation appears to be working correctly.

To generate this message, Docker took the following steps:
 1. The Docker client contacted the Docker daemon.
 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
    (amd64)
 3. The Docker daemon created a new container from that image which runs the
    executable that produces the output you are currently reading.
 4. The Docker daemon streamed that output to the Docker client, which sent it
    to your terminal.

To try something more ambitious, you can run an Ubuntu container with:
 $ docker run -it ubuntu bash

Share images, automate workflows, and more with a free Docker ID:
 https://hub.docker.com/

For more examples and ideas, visit:
 https://docs.docker.com/get-started/
```